src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -50,6 +52,8 @@ public static Name parseName(String name) throws ParseException {
         return new Name(trimmedName);
     }
 
+
+
```

Comment:
Nitpick: Perhaps you can consider removing the extra indentations

Reply:
👍